docs/source/install.rst

```diff
@@ -1,15 +1,51 @@
 Install Dask
 ============
 
-You can install dask using ``conda``::
+Dask is available to Anaconda users via either the conda or pip commands, both of which are included in Anaconda. If you do not already have Anaconda, `download Anaconda now <https://www.continuum.io/downloads>`_.
+
+Dask can then be installed using ``conda``::
 
     conda install dask
 
-You can install dask using ``pip``::
+Or dask can be installed with ``pip``::
 
     pip install dask
 
-Dask also ships with Anaconda so you may have it already.
+
+Test dask install
+-----------------
+
+Because dask is a Python library, rather than a graphical program or command line program, to test that dask is working you can run a small Python program. 
+
+In this sample test, you will create a conda environment and name it dasktest. Activate the environment, then at the command line type `python` to start an interactive Python shell, run a brief set of commands to display an array of random numbers, then quit and deactivate the dasktest environment:
+
+.. code::
+
+   conda create -n dasktest dask
+   source activate dasktest
+   python
+
+.. code::
+
+   import dask.array
+   import numpy
+   a=dask.array.random.normal(6,.1,size=(4,4),chunks=(2,2))
+   numpy.array(a)
+   quit()
+
+.. code::
+
+   source deactivate
+
+If dask is installed correctly, this will print out a 4x4 array of random numbers close to 6, similar to:
+
+.. code::
+
+   >>>array([[ 6.11636086,  6.02888376,  6.02715362,  6.09789198],
+       [ 6.17750019,  6.10241887,  6.0984278 ,  5.80852248],
+       [ 6.00780242,  6.03833159,  6.0115587 ,  6.06790745],
+       [ 5.9882688 ,  5.91056546,  5.9793473 ,  5.93219086]])
+
```

Comment:
I'm unlikely to work on this in the next couple of weeks.  If others want
to take it over that'd be appreciated.  The various notebooks in the
dask-examples repository may be of some limited use.

On Thu, Jan 7, 2016 at 2:36 PM, Will Warner <notifications@github.com>
wrote:

> In docs/source/install.rst
> <https://github.com/blaze/dask/pull/908#discussion_r49136426>:
>
> > +   numpy.array(a)
> > +   quit()
> > +
> > +.. code::
> > +
> > +   source deactivate
> > +
> > +If dask is installed correctly, this will print out a 4x4 array of random numbers close to 6, similar to:
> > +
> > +.. code::
> > +
> > +   >>>array([[ 6.11636086,  6.02888376,  6.02715362,  6.09789198],
> > +       [ 6.17750019,  6.10241887,  6.0984278 ,  5.80852248],
> > +       [ 6.00780242,  6.03833159,  6.0115587 ,  6.06790745],
> > +       [ 5.9882688 ,  5.91056546,  5.9793473 ,  5.93219086]])
> > +
>
> I think that is an excellent idea! If you want to do it in bash and save
> it to a file with 'script', or something like that, I can flesh out the
> prose and turn it into a set of small quickstart pages. Our new docs
> contractor Geri Ochoa @gochoam <https://github.com/gochoam> is also
> reading up on dask, and he may be able to help with this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blaze/dask/pull/908/files#r49136426>.
>